medical_prescription/models/medical_prescription_order_line.py

```diff
@@ -27,6 +27,16 @@ class MedicalPrescriptionOrderLine(models.Model):
     qty = fields.Float(
         string='Quantity',
     )
+    name = fields.Char(
+        required=True,
+        default=lambda s: s._default_name(),
```

Comment:
@jcdrubay I'm not entirely sure to be honest. I can see the logic in making it readonly, however I think there may be situations where it may need to be changed or corrected. I double in sale to compare, and noticed that sale_order_line names are not required. Perhaps @lasley can chime in.